### PR TITLE
PMI-587 RP report script improvements and documentation

### PR DESCRIPTION
**Pull request by donnabelsey**
Minor changes actioning the feedback from the PA's when running the script for the first time. Includes:
- Write reports into a separate folder for each start date
- Rename csv files to format `{startdate}-{rp-name}-rp_report.csv`
- Add name for All-RPs csv file
- Rename virtualenv to indicate which repo it belongs to
- Rearrange Readme into step-by-step instructions for running the RP Report

Also deletes the dummy.txt file, leftover from when we were testing the upload-to-s3 script.

Additional context on the reason for some of these changes is in the commit messages.

### CODE REVIEW

File: performance/reports/rp.py [hunk 1]
```diff
@@ -42,14 +42,16 @@ def get_df_successes_by_rp(df_verifications_by_rp):
 
 
 def export_metrics_to_csv(df_export, report_output_path, date_start):
-    if not os.path.exists(report_output_path):
-        os.makedirs(report_output_path)
+    report_output_path_for_week = os.path.join(report_output_path, date_start)
```
Comment (galund) [inline performance/reports/rp.py]: Can we have `output/rp_report/<date_start>` and lose the prefix/postfix from the file names maybe?

File: performance/reports/rp.py [hunk 2]
```diff
@@ -42,14 +42,16 @@ def get_df_successes_by_rp(df_verifications_by_rp):
...
+    if not os.path.exists(report_output_path_for_week):
+        os.makedirs(report_output_path_for_week)
+
     # Create export file with all RPs data
-    df_export.to_csv(os.path.join(report_output_path, f'rp_report-{date_start}.csv'))
+    df_export.to_csv(os.path.join(report_output_path_for_week, f'{date_start}-_All-RPs-rp_report.csv'))
```
Comment (galund) [inline performance/reports/rp.py]: I reckon the capitalisation is a bit ugly myself, and if the date is in the dirname then maybe we don't need the date in the filename too? So perhaps just `_all.csv` if we're also going with the above suggestion re dirnames.

File: setup.cfg [hunk 3]
```diff
@@ -1,2 +1,2 @@
 [tool:pytest]
-norecursedirs = venv* src
\ No newline at end of file
+norecursedirs = *venv* src
```
Comment (galund) [inline setup.cfg]: I wonder why `src` is in here at all? Seems weird

File: .gitignore [hunk 4]
```diff
@@ -8,6 +8,7 @@ __pycache__
 
 .env
 verify-performance-scripts-env
```
Comment (galund) [inline .gitignore]: Hmm wonder what this is?

File: performance/reports/rp.py (hunk 1 above)
Comment (donnabelsey) [reply performance/reports/rp.py]: I'll add a top-level folder for RP-Reports. The requirement for the filenames came from the PA's and seeing as they're the primary user for this I think that's probably not negotiable.

File: .gitignore [hunk 5]
```diff
@@ -1,13 +1,12 @@
 # generated RP reports
 output/
 
-venv*
+verify-performance-scripts-venv
```
Comment (galund) [inline .gitignore]: Of course now all the other places use `*venv*`... :-D

**Pull request merged**